File: 1-easy/problems/3392-count-subarrays-of-length-three-with-a-condition/solution.ts
function countSubarrays(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return sum((nums[i-1]+nums[i+1])*2 == nums[i] for i in Array.from({length: 1, (nums}, (_, i) => i).length-1));
}

export { countSubarrays };

File: 1-easy/problems/3392-count-subarrays-of-length-three-with-a-condition/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def countSubarrays(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum((nums[i-1]+nums[i+1])*2 == nums[i] for i in range(1, len(nums)-1))
